packages/opentelemetry-tracing/src/NoopSpanProcessor.ts

```diff
@@ -21,6 +21,10 @@ import { ReadableSpan } from './export/ReadableSpan';
 export class NoopSpanProcessor implements SpanProcessor {
   onStart(span: ReadableSpan): void {}
   onEnd(span: ReadableSpan): void {}
-  shutdown(): void {}
-  forceFlush(): void {}
+  shutdown(cb: () => unknown = () => {}): void {
+    setTimeout(cb, 0);
```

Comment:
Why `setTimeout` vs `setImmediate`? To support older runtimes?

Reply:
I originally had `setTimeout` but the browser test failed.